src/pages/ProfilePage/ProfilePage.jsx

```diff
@@ -93,10 +94,12 @@ const ProfilePage = () => {
             const userRef = doc(firestore, `users/${userID}`);
 
             let response1 = await getDoc(userRef);
```

Comment:
If variable is not re initialized, prefer using const

Reply:
okay